src/controllers/__tests__/user.test.js

```diff
@@ -0,0 +1,194 @@
+const request = require('supertest');
+
+const server = require('../../app');
+
+jest.setTimeout(5000);
+
+const {
+  closeDatabase,
+  clearDatabase,
+  connectToMongo,
+} = require('../../db/connection');
+
+let userId;
+
+const invalidId = 'invalidId123';
+
+const notExistingUserId = '937c7f79bcf86ce7863f5d0a';
+
+const newValidUser = {
+  username: 'new.user',
+  firstName: 'New',
+  lastName: 'User',
+  email: 'email@domain.com',
+  password: 'testPassword',
+  password2: 'testPassword',
+  address: 'new address',
+};
```

Comment:
In a previous merged PR, there are some fictional names used for mock user inputs. Once you merge these changes you can change your inputs also to fictional names and emails.